docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+# mLRS Documentation: Quick Start Guide #
+
+([back to main page](../README.md))
+
+This page provides hints, suggestions and links to help get new users started with mLRS quickly.
+
+This guide covers mLRS in its core use case to provide a single radio link to carry both RC control and full bi-directional MAVLink Telemetry.  When we discuss configuration of your flight controller, we assume you are running ArduPilot.  We also assume you use OpenTX or EdgeTX and that you will connect a ground station and/or use the [Yaapu Telemetry App](CRSF.md).
+
+The steps here should be followed in order and when practical we provide a way to confirm a step was completed successfully.
+
+We expect that you are starting fresh and that all ArduPilot and mLRS parameters not covered here still have their default settings.
+
+Some notation and terminology:
+- We use "Tx module" and "receiver" to refer to a mLRS module in the Tx/transmitter role and Rx/receiver role. In the use case assumed here, the Tx module will be installed in a OpenTX or EdgeTX radio handset, and the receiver will be mounted on the vehicle and connected to the flight controller.
+- We use lower case "rx" and "tx" when referring to the receive and transmit pads or pins of a serial port connection.
+
+## 1. Choose and purchase a Tx module and a compatible receiver
+
+There are many supported choices of mLRS hardware, including off-the-shelf options and  many DIY designs.
+We cover here three popular but mutually incompatible commercially available off-the-shelf options.  We summarize some pros and cons below to help you choose. Don't mix Rx and Tx between these options.
+
+### MatekSys mLRS [mR900-30-TX](https://www.mateksys.com/?portfolio=mr900-30-tx) module kit with their [mR900-30](https://www.mateksys.com/?portfolio=mr900-30) receiver
+- These 900 MHz band modules were designed specifically for mLRS and arrive pre-flashed with mLRS version 1.3.
+- This option should provide the longest range of the three discussed here.
+- The mR900-30-TX module kit includes an HC04 module for a convenient Bluetooth serial connection to your ground station software running on a computer or Android device.
+- These modules are especially easy to flash via DFU over their USB C connection.
+- These use the SX126x LoRa chip and support 19 Hz, 31 Hz and FSK(50 Hz) modes.
+- This option does not work with any ELRS receivers.
+- The Tx module kit does not include an antenna or a JR bay case which must be 3D printed.  If you don't have access to a 3D printer, you can purchase a print from a number of 3D printing services.
+- The [mR900-22](https://www.mateksys.com/?portfolio=mr900-22) is a compatible lower power receiver which might be appropriate for smaller vehicles.  But it is not yet recommended for new users because it does not include USB for flashing and improvements to ArduPilot serial passthrough to support firmware update without disconnecting the receiver from the flight controller are not yet released.
```

Comment:
I frankly don't think a NEW USER should be burden with that much chit chat ... 

"But it is currently not recommended for new users because it does not include USB for flashing." Period.

Reply:
I'll simplify this, but the issue is not the lack of USB.  That's a feature contributing to light weight and small size.  The problem is ArduPilot passthrough and lack of automated tools to use it.

Reply:
IMHO it IS a lack of DFU ... all the other methods are IMHO a long mile away from being nearly as friendly as DFU ...

Reply:
From my perspective, that is only because the web application only supports DFU so far.  If the web app supported flashing via passthrough and had the same automation to setup the passthrough and start the bootloader that the python script has, it would be almost exactly as easy as DFU.  One extra box to fill in to to specify the FC serial port would be the only difference.

Reply:
Anyway, how does the new text look?